### ISSUE TITLE
[ENG-872] Pressing enter on selected file does the same action as double click

### ISSUE THREAD

**Issue by utkubakir** [spacedriveapp/spacedrive#1148]
* Pressing enter on a highlighted file or directory should open it and be the same as double clicking.

<sub>From [SyncLinear.com](https://synclinear.com) | [ENG-872](https://linear.app/spacedriveapp/issue/ENG-872/pressing-enter-on-selected-file-does-the-same-action-as-double-click)</sub>

**Comment by PineappleRind** [comment]
in Finder pressing Enter on a selected file/directory allows renaming, is this a Windows thing?

**Comment by utkubakir** [comment]
> in Finder pressing Enter on a selected file/directory allows renaming, is this a Windows thing?

Good point, I think the default behavior for Windows is to open it so we should also do this as well.
